mobile-app/app/screens/AppNavigator/screens/Portfolio/LOCK/LockTransactions.tsx

```diff
@@ -0,0 +1,329 @@
+        option: {
+          header: () => {
+            return (
+              <ThemedView
+                light={tailwind('bg-white border-0')}
+                dark={tailwind('bg-white border-0')}
+                style={tailwind('flex flex-row justify-between items-center px-4 py-2 border-b', {
+                  'py-3.5 border-t -mb-px': Platform.OS === 'android',
+                })} // border top on android to handle 1px of horizontal transparent line when scroll past header
+              >
+                <Text style={tailwind('text-base font-medium text-black')}>
+                  {translate('LOCK/LockDashboardScreen', 'Note to the transaction')}
+                </Text>
+                <TouchableOpacity onPress={() => dismissModal()}>
+                  <ThemedIcon
+                    iconType="MaterialIcons"
+                    name="close"
+                    size={20}
+                    light={tailwind('text-black')}
+                    dark={tailwind('text-black')}
+                  />
+                </TouchableOpacity>
+              </ThemedView>
+            );
+          },
+          headerBackTitleVisible: false,
+        },
+      },
+    ]);
+    expandModal();
+  }
+
+  return (
+    <View style={tailwind('flex-1 bg-white', { 'justify-center': isLoading })}>
+      {isLoading ? (
+        <Loading message={translate('LOCK/LockTransactions', 'Loading all transactions. Please wait...')} lock />
+      ) : (
+        <>
+          <TransactionTypeSelector current={selectedType} onChanged={setSelectedType} />
+          <ThemedFlatList
+            data={transactions
+              ?.filter((t) => (selectedType ? t.type === selectedType : true))
+              .filter((t) =>
+                selectedType === TransactionType.REWARD
+                  ? activeTab === RewardType.PAID_OUT
+                    ? t.status !== TransactionStatus.WAITING_FOR_BALANCE
+                    : t.status === TransactionStatus.WAITING_FOR_BALANCE
+                  : true,
+              )
+              .filter((_, i) => i < (page + 1) * pageSize)}
+            renderItem={({ item }: { item: TransactionDto }): JSX.Element => (
+              <TransactionRow transaction={item} openInfo={openInfo} />
+            )}
+            keyExtractor={(item, i) => `${item.type}-${i}`}
+            ListHeaderComponent={
+              selectedType === TransactionType.REWARD ? (
+                <View style={tailwind('px-4 py-2 bg-white')}>
+                  <ButtonGroup buttons={buttonGroup} activeButtonGroupItem={activeTab} testID="dex_button_group" lock />
+                  <InfoText
+                    text={translate(
+                      'LOCK/LockDashboardScreen',
+                      'Please note that we will collect and pay out all amounts below 0.0001 for tokens in the corresponding token assets for you as soon as the value is >= 0.0001. For Liquidity Pool tokens this value equals 1$',
```

Comment:
Please note that only amounts >0.0001 will be paid out directly, otherwise accumulated until the amount is >0.0001 in the respective assets. For liquidity pool tokens this value equals 1$.